### PR TITLE
Add additional color for error image

### PR DESCRIPTION
**Pull request by jjeroch**
add additional color settings

### CODE REVIEW

**Comment by oyo** [comment]
@jjeroch oh je - Duplication hatten wir nicht berücksichtigt. Du kannst es im Sonarcloud auf `won't fix` oder so stellen

**Comment by jjeroch** [comment]
> @jjeroch oh je - Duplication hatten wir nicht berücksichtigt. Du kannst es im Sonarcloud auf `won't fix` oder so stellen

Poor me :D ;) aber ein Glück das es "Wont fix" gibt.

**Comment by jjeroch** [comment]
:/ wont fix or descope is not possible. @burakbengi  I have seen that you have admin rights in Sonarcloud, can you help out to get the validation on this file disabled. Off course we should reduce the duplication, but I would expect that this should not restrict us right now. What do you think?

**Comment by oyo** [comment]
@jjeroch I recently did something very similar with the Logo. If we save both the svg's as files then that won't be considered duplication: https://github.com/catenax-ng/product-portal-frontend/blob/main/cx-portal-shared-components/src/components/basic/Logo/index.tsx

File: cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorImage.tsx [hunk 1]
```diff
@@ -679,5 +680,400 @@ export const ErrorImage = () => {
         </radialGradient>
       </defs>
     </svg>
+  ):
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorImage.tsx:683]: why does this line look so sad? :(

**Comment by burakbengi** [comment]
Before I meddle with Sonarcloud Settings maybe @devsecops-catenax can disable sonar check on .svg files? @carslen @Anacin @SebastianBezold

**Pull request closed without merging**